### ISSUE TITLE
Command Prompt crashes when I put token in.

### ISSUE THREAD

**Issue by editsellar** [slow/account-ruiner#2]
When I put the token into the command prompt it says logging in, then it closes the command prompt and doesn't work, is there any way to fix this?

**Comment by papseryouknow** [comment]
yeah, he's right, would be cool to get fixed. the ruiner was awesome.
